### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by e-kwsm**

### CODE REVIEW

File: messageDir/en.lua [hunk 1]
```diff
@@ -174,7 +174,7 @@ Alternatively, you can set the environment variable LMOD_DISABLE_SAME_NAME_AUTOS
 for available versions.]==],
      e_Spdr_Timeout        = "Spider search timed out.\n",
      e_Swap_Failed         = "Swap failed: \"%{name}\" is not loaded.\n",
-     e_Unable_2_Load       = "Unable to load module because of error when evaluating modulefile: %{name}\n     %{fn}: %{message}\n     Please check the modulefile and especially if there is a the line number specified in the above message",
+     e_Unable_2_Load       = "Unable to load module because of error when evaluating modulefile: %{name}\n     %{fn}: %{message}\n     Please check the modulefile and especially if there is the line number specified in the above message",
```
Comment (boegel) [inline messageDir/en.lua:177]: "`if there is a line number`" is probably better?

**Review by boegel — COMMENTED** [review]
Very nice contribution @e-kwsm! Just one minor suggested change...

File: messageDir/en.lua (hunk 1 above)
Comment (e-kwsm) [reply messageDir/en.lua]: Fixed.

File: tools/ColumnTable.lua [hunk 2]
```diff
@@ -1,5 +1,5 @@
 --------------------------------------------------------------------------
--- This class is takes a array of strings and produced a multi-column output.
+-- This class takes a array of strings and produced a multi-column output.
```
Comment (e-kwsm) [inline tools/ColumnTable.lua:2]: The expression `a array` appears in some files, and actually this file has a variable `a`, but perhaps they should be `an array`.

**Pull request merged**